### PR TITLE
update to 2017

### PR DESCRIPTION
**Pull request by qwIvan**

### CODE REVIEW

File: he/currentYear [hunk 1]
```diff
@@ -1 +1 @@
-5774
+5775
```
Comment (asgrim) [inline he/currentYear]: Does Hebrew calendar change at same time as Gregorian calendar?

File: zh-cn/currentYear [hunk 2]
```diff
@@ -1 +1 @@
-4714
+4715
```
Comment (asgrim) [inline zh-cn/currentYear]: Does Chinese calendar change at same time as Gregorian calendar?

Comment (qwIvan) [inline zh-cn/currentYear]: no。。。

File: he/currentYear (hunk 1 above)
Comment (qwIvan) [inline he/currentYear]: no。。。

**Pull request closed without merging**